lib/supported-assertions.js

```diff
@@ -1,14 +1,12 @@
 'use strict';
 
-module.exports = {
-  'equal': 'binary',
-  'notEqual': 'binary',
-  'deepEqual': 'binary',
-  'notDeepEqual': 'binary',
-  'propEqual': 'binary',
-  'notPropEqual': 'binary',
-  'strictEqual': 'binary',
-  'notStrictEqual': 'binary',
-  'ok': 'unary',
-  'notOk': 'unary'
-};
+export const equal = 'binary';
```

Comment:
Any idea how this lib is used? This would be classed as a breaking change if it is public API? Any particular reason why it was refactored like this?